### PR TITLE
Add wrapper for qfilt.

### PR DESCRIPTION
**Pull request by davebx**
This re-adds the qfilt wrapper from #1021

### CODE REVIEW

File: tools/qfilt/qfilt.xml [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+                <param name="mismatch" argument="-t" type="integer" value="0" label="Mismatch" help="if PREFIX is supplied, prefix matching tolerates at most MISMATCH mismatches" />
+                <param name="format" argument="-f" type="select" label="Format" help="output in FASTA or FASTQ format (default=FASTA)" >
+                    <option value="FASTA">FASTA</option>
+                    <option value="FASTQ">FASTQ</option>
+                </param>
+            </when>
+        </conditional>
+    </inputs>
+    <outputs>
+        <data format="fasta" name="filtered_output"/>
```
Comment (bgruening) [inline tools/qfilt/qfilt.xml]: You should change the format depending on the format argument which you are asking from the user.

File: tools/qfilt/qfilt.xml [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    </tests>
+    <help><![CDATA[
+qfilt
+=====
+
+This simple program is meant to filter sequencing data,
+optionally removing or splitting reads with poor quality scores
+and to optionally _only_ retain fragments from reads that are tagged with a given 5' sequence.
+
+### OUTPUT: ####
```
Comment (bgruening) [inline tools/qfilt/qfilt.xml]: The output is missing here? Is the rest about stderr and stdout intended?

Comment (davebx) [reply tools/qfilt/qfilt.xml]: Yeah, I'll change that to `### OUTPUTS: ####`

File: tools/qfilt/qfilt.xml [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+qfilt
+=====
+
+This simple program is meant to filter sequencing data,
+optionally removing or splitting reads with poor quality scores
+and to optionally _only_ retain fragments from reads that are tagged with a given 5' sequence.
+
+### OUTPUTS: ####
+
+#### stderr: ####
```
Comment (bgruening) [inline tools/qfilt/qfilt.xml]: @davebx I was more concerned about the stderr and stdout. Is this really just accessible on stdout? Most of the users will not now what stderr means. Is the stderr really needed?
Can we not simply write, example output ... fasta content ... done?

Also `#### foo ####` is not valid rst, isn't it?

**Pull request merged**